README.md

```diff
@@ -227,6 +240,16 @@ documentation.
 
 Check the `--min-acl` setting -- see [above](#controlling-what-is-documented).
 
+**Unable to find an Xcode with swift version X**
+
+1. The value passed with `--swift-version` must exactly match the version
+   number from `swiftc --version`.  For example Xcode 10.1 needs
+   `--swift-version 4.2.1`.  See [the flag documentation](#choosing-the-swift-language-version).
+2. The Xcode you want to use must be in the Spotlight index.  You can check
+   this using `mdfind 'kMDItemCFBundleIdentifier == com.apple.dt.Xcode'`.
+   Some users have reported this issue being fixed by a reboot; `mdutil -E`
+   may also help.
```

Comment:
May want to say that if the Xcode isn't tin the Spotlight index, another alternative is to specify it via `DEVELOPER_DIR`

Reply:
OK